### PR TITLE
feat(QTM-469): Added property to adjust Card background opacity

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-469

## Review guide
- [ ] Unit tests
- [ ] Regression
- [ ] Code review
- [ ] Documentation
- [ ] TypeScript updated

### CODE REVIEW

File: components/Card/alphaNumberPropTypesValidator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const isValidAlphaNumber = value => {
```
Comment (rapahaeru) [inline components/Card/alphaNumberPropTypesValidator.js:1]: image

image

 I think the test simulating error is missing.
WDYT? Make it now or generate a technical  debit?

File: components/Card/Card.jsx [hunk 2]
```diff
@@ -59,12 +67,15 @@ Card.propTypes = {
   theme: PropTypes.shape({
     colors: PropTypes.object,
   }),
+  /** alpha value (number in range [0,1]) to set background opacity. 0 is fully transparent and 1 is fully opaque. */
```
Comment (rapahaeru) [inline components/Card/Card.jsx:70]: perfect!

File: stories/Card/Card.regression-test.story.jsx [hunk 3]
```diff
@@ -8,8 +8,18 @@ export default {
   component: Card,
 };
 
+const Wrapper = ({ children }) => (
+  <div style={{ backgroundColor: '#1250C4', padding: 8 }}>{children}</div>
```
Comment (rapahaeru) [inline stories/Card/Card.regression-test.story.jsx:12]: nice!

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [!] Unit tests
- [x] Regression
- [x] Code review
- [x] Documentation
- [x] TypeScript updated

File: components/Card/alphaNumberPropTypesValidator.js (hunk 1 above)
Comment (MarcosViniciusPC) [reply components/Card/alphaNumberPropTypesValidator.js:1]: It went unnoticed by me, I'll do it myself, I don't see any problems.

File: components/Card/alphaNumberPropTypesValidator.unit.test.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import alphaNumberPropTypesValidator from './alphaNumberPropTypesValidator';
```
Comment (rapahaeru) [inline components/Card/alphaNumberPropTypesValidator.unit.test.js:1]: woooooow

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**